book/website/reproducible-research/renv/renv-containers.md

```diff
@@ -420,6 +430,120 @@ docker run -d --name dind-rootless --privileged docker:20.10-dind-rootless
 
 The following prequisites, which are part of the [`shadow-utils`](https://github.com/shadow-maint/shadow) package are required to run Docker rootless: `newuidmap` and `newgidmap`.
 
+(rr-renv-containers-podman)=
+## Podman
+
+[Podman](https://podman.io/) is an open-source container engine that can be seen as a compelling alternative to Docker.
+In fact, Podman provides a Docker-compatible command line interface.
+For most use cases a user familiar with Docker can simply alias the Docker command to Podman (`alias docker=podman`) and carry on as normal.
+Podman includes a [full set of tools](https://docs.podman.io/en/latest/Commands.html) to create, run and manage and share containers.
+
+Podman is free and open-source software released under the Apache License 2.0.
+This is in contrast to Docker which has some open-source components, such as the engine and command line interface, but also develops closed-source, [subscription-requiring](https://www.docker.com/blog/updating-product-subscriptions/) software including the Docker Desktop clients for MacOS and Windows.
+
+All [Open Container Initiative](https://opencontainers.org/) container images can be used with Podman including Docker images hosted on Docker Hub.
+It is likely existing projects using Docker can be migrated to Podman.
+
+Unlike Docker which uses a daemon running as root, Podman is daemonless.
+Unprivileged users can run containers using Podman.
+In most cases this will be configured automatically.
+This avoids a problem with the standard Docker configuration where users able to run containers have implicit access to the Docker daemon.
```

Comment:
```suggestion
This avoids a problem with the standard Docker configuration where users can run containers with an implicit access to the Docker daemon.
```

Reply:
I think this changes the meaning of the sentence.

I want to explain how in the 'normal' Docker setup adding a user to the docker group is essentially giving them a high-level of access to the host with a very simple route to privilege escalation. For example,

```
docker run --mount=type=bind,source=/,destination=/host -it busybox
```

Maybe it would be worth explaining that properly instead of alluding to it?

Reply:
Thank you Jim, for the clarification! I think it's only me who couldn't understand it, English isn't my 1st language but I think you can ignore this comment - I think I can remove the suggestion from my side.